spec/models/user_project_spec.rb

```diff
@@ -0,0 +1,26 @@
+# == Schema Information
+#
+# Table name: user_projects
+#
+#  id         :bigint(8)        not null, primary key
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  project_id :bigint(8)
+#  user_id    :bigint(8)
+#
+# Indexes
+#
+#  index_user_projects_on_project_id  (project_id)
+#  index_user_projects_on_user_id     (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (project_id => projects.id)
+#  fk_rails_...  (user_id => users.id)
+#
+
+require 'rails_helper'
+
+RSpec.describe UserProject, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
add spec to check something like

expect(create(:user_project)).to be_persisted